.teamcity/settings.kts

```diff
@@ -42,7 +42,7 @@ To debug in IntelliJ Idea, open the 'Maven Projects' tool window (View
 'Debug' option is available in the context menu for the task.
 */
 
-version = "2021.1"
+version = "2022.04"
```

Comment:
If we look above at the imports, we're importing like this:

`import jetbrains.buildServer.configs.kotlin.v2019_2.buildFeatures.pullRequests`. I think we should probably update the versions in that string as well at some point, but I don't really know if it would make a difference

Reply:
I assume you're specifically referring to the "2019" part in the import, correct?

I don't know if that would make a difference either. We've been on 2021.1 for quite some time and the imports still work fine.

Reply:
right, exactly